pandas/core/indexes/base.py

```diff
@@ -4167,6 +4169,37 @@ def invalid_op(self, other=None):
 Index._add_comparison_methods()
 
 
+def dispatch_missing(op, left, right, result):
+    """
+    Fill nulls caused by division by zero, casting to a diffferent dtype
+    if necessary.
+
+    Parameters
+    ----------
+    op : function (operator.add, operator.div, ...)
+    left : object, usually Index
```

Comment:
usually Index means?

Reply:
That the input `left` is going to be an Index object most of the time, but _could_ be an arbitrary object.

Reply:
I know what it means, I am asking to have you elaborate in the note on these cases, IOW why would be it be an arbitrary object, eg.. when the op is reversed?